<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CmainWindow</class>
 <widget class="QMainWindow" name="CmainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Re-Typeset</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="srcDirLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Input directory:</string>
    </property>
   </widget>
   <widget class="QLabel" name="destDirLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Output directory:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="srcDir">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>411</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="destDir">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>411</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="srcDirButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="destDirButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="pageSize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Size:</string>
    </property>
    <widget class="QLabel" name="pageSizeHeightLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Height:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="pageSizeHeight">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
     </property>
     <property name="suffix">
      <string notr="true"/>
     </property>
     <property name="minimum">
      <number>99</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>800</number>
     </property>
    </widget>
    <widget class="QLabel" name="pageSizeWidthLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Width:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="pageSizeWidth">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
     </property>
     <property name="suffix">
      <string notr="true"/>
     </property>
     <property name="minimum">
      <number>99</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>600</number>
     </property>
    </widget>
    <widget class="QLabel" name="pageSizeMarginLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Margin:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="pageSizeMargin">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
     </property>
     <property name="suffix">
      <string notr="true"/>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="value">
      <number>15</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="tresh">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Treshold</string>
    </property>
    <widget class="QSpinBox" name="treshVal">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>254</number>
     </property>
     <property name="value">
      <number>158</number>
     </property>
    </widget>
    <widget class="QLabel" name="treshLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Value:</string>
     </property>
    </widget>
    <widget class="QSlider" name="treshSlider">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>254</number>
     </property>
     <property name="value">
      <number>158</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="outProgressBar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>611</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="textSize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Text height:</string>
    </property>
    <widget class="QLabel" name="textSizeHeightLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Line height:</string>
     </property>
    </widget>
    <widget class="QLabel" name="textLinesPerPageLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Lines per page:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="textSizeHeight">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="value">
      <number>23</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="textLinesPerPage">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>27</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="outStartButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>611</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QGroupBox" name="options">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Additional options:</string>
    </property>
    <widget class="QCheckBox" name="optionsHardMargins">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Hard margins</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="optionsNoUpscaling">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>No upscaling</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="optionsFullColorOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Full color output</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="optionsJustify">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>89</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Justify</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="optionsComicMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Comic mode</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="optionsRotateImages">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate images</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="optionsFindDividedWords">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Find divided words</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="outFiles">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Output files names:</string>
    </property>
    <widget class="QLabel" name="outFilesLabel">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>_NUM.png</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="outFilesPrefix">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>521</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="predef">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>151</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Automatic size:</string>
    </property>
    <widget class="QPushButton" name="predefScreen1440">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1080×1440, 6.8&quot;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="predefScreen1024">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>758×1024, 6&quot;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="predefScreen800">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>600×800, 6&quot;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="predefLines15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>15 lines</string>
     </property>
    </widget>
    <widget class="QPushButton" name="predefLines20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>20 lines</string>
     </property>
    </widget>
    <widget class="QPushButton" name="predefLines25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>25 lines</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="destDirCreate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>71</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Create directory NAME.out</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>&amp;Language</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionShowDebugImages"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuLanguage"/>
  </widget>
  <action name="fileOpen">
   <property name="text">
    <string>Otwórz</string>
   </property>
  </action>
  <action name="actionShowDebugImages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show debug images</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About application</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
